src/main/webapp/javascript/main.js

```diff
@@ -28,4 +27,27 @@ function logout() {
 }
 function goMyProfile() {
     location.href = "/profile.jsp";
+}
+function join(button) {
+    var data = {};
+    var places = $(button).closest('tr').find('.table_places').text() - 1;
+    var tripId = $(button).closest('tr').find('.table_id').text();
+    var id = $(button).closest('td').parent().index();
+    var x = document.getElementById("trips").rows[id + 1].cells;
+    x[4].innerHTML = places;
```

Comment:
The variable name `x` does not explain its purpose. Please use more meaningful names.
E.g. `places` and `tripId` are OK names, while `id` does not make it clear whose id it is, and `x` is an unknown.